src/toolkit/gis.h

```diff
@@ -0,0 +1,158 @@
+#ifndef CYCLUS_SRC_GIS_H_
+#define CYCLUS_SRC_GIS_H_
+
+#include <math.h>
+#include <stdio.h>
+#include <sstream>
+#include <string>
+
+#include "cyclus.h"
+
+// forward declarations
+namespace cyclus {
+namespace toolkt {
+class GIS;
+}
+}
+
+namespace cyclus {
+namespace toolkit {
+/// @class GIS
+/// The GIS class is a basic class that stores the geographic location of each
+/// agent in latitude and longitude and follows the ISO
+/// 6709 standard. Allows string expression of GIS objects as specified in
+/// ANNEX H of ISO 6709 found here: 'https://www.iso.org/standard/39242.html'
+/// or a quick simplified version found here
+/// 'https://en.wikipedia.org/wiki/ISO_6709'
+///
+/// Brief Description of ISO 6709:
+/// 1. Latitude comes before Longitude
+/// 2. North latitude is positive
+/// 3. East longitude is positive
+/// 4. Fraction of degrees (decimal values) is prefered in digital data
+/// exchange.
+///
+/// The class is adapted from 'https://github.com/jaime-olivares/coordinate'
+/// under the MIT License.
```

Comment:
The MIT license requires the copyright notice to be included, specifically here `Copyright (c) 2015 Jaime Olivares`